distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2301,6 +2301,15 @@ Event event = new Event("ddf/notification/catalog/downloads", properties);
 eventAdmin.postEvent(event);
 ----
 
+=== Configuring Thread Pools
+
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through system properties at the beginning of Karaf's boot process. The `org.codice.ddf.system.threadPoolSize` property can be used to specify the size of thread pools used by:
+* Federating requests between {branding} systems
+* Downloading resources
+* Handling asynchronous queries, such as queries from the UI
+
```

Comment:
Suggestion: "This value is set to 128 thread by default. **Since the operating system has a limited number of threads available and each thread consumes memory, setting this value too high can cause the system to run out of threads or memory and fail.** If unsure, leave this setting at **its** default value of 128".